src/foundation.cs

```diff
@@ -6264,7 +6264,7 @@ partial interface NSUrl : NSSecureCoding, NSCopying
 		[Field ("NSURLIsApplicationKey")]
 		NSString IsApplicationKey { get; }
 
-		[iOS (9,0)][NoMac]
+		[iOS (9,0), Mac(10,11)]
```

Comment:
10,11??? Was this missed from other years, or a typo?

Reply:
Headers say `macos(10.11)` but makes sense that it could be a typo from Apple

```suggestion
		[iOS (9,0), Mac(11,0)]
```